packages/compartment-mapper/src/archive-lite.js

```diff
@@ -0,0 +1,475 @@
+// @ts-check
```

Comment:
tbh I'd love to see a 
```js
/**
 * This module is for doing x, y, z and it is here for reasons.
 * @module
 */
```
at the top of each file.

the subtext being that I'm not sure how these low-calorie modules differ.

not really getting a great idea for how much duplication is going on, either. but that's just cause GH's PR UI won't let me look at two different files side-by-side at the same time.

Reply:
I'm wondering if it wouldn't be unreasonable to rename some of these subsystems (modules, functions) to something other than `*archive*` if they don't actually interface with archive files (or, generally speaking, just draw the line somewhere else).  I'm using some of the `*archive*`-named API surface just to get at `Sources`.  naming things is hard

Reply:
I’ll circle back and add file docs.

I think it would be reasonable to factor out an API for the leading half of `archive.js` that doesn’t entrain a dependency on Zip.

The thing that makes these `-lite` is that they do not import any parsers, which in turn might entrain Babel.